content/docs/git/ide.md

```diff
@@ -0,0 +1,248 @@
++++
+title = "30/September - Using an IDE (Integrated development environment)"
+description = "Goals"
+# weight = 10
++++
+
+![VSCode](https://media0.giphy.com/media/SS8CV2rQdlYNLtBCiF/giphy.gif)
+
+## Prerequisites
+
+### 1. Install VsCode
+
+Visual Studio Code is a freeware source-code editor made by Microsoft for Windows, Linux and macOS. Features include support for debugging, syntax highlighting, intelligent code completion, snippets, code refactoring, and embedded Git. Wikipedia
+
+During this course, while working with HTML and Pything we might end up working with an code editor to make easier interacting with files and folder. An editor can be considered the notebook of a programmer. This, with the terminal, will be considered your main tools to reach your goals :D
+
+To install VSCode please download it by clicking on the Download button of the next link ->
+
+[VSCode](https://code.visualstudio.com/)
+
+You can start playing with it if you want, during the class we will see how to work with it.
+
+### 2. Review exercises from Tuesday (21/September)
+
+Review the content from last class. Make sure that have completed the prerequisites from last class, including that your email is verified on GitHub. If you get stuck or feel confused about anything, no worries! Please write down any questions you have so that we can go over these in class.
+
+{{< button "../git_and_github/" "Review Previous Session: 21/September" >}}
+
+---
+## Class Curriculum
+
+| Section content                          | Expected time (mins) | Pre - Requirements |
+| ---------------------------------------- | -------------------- | ------------------ |
+| Little quiz with Kahoot  | 10        | ❌             |
```

Comment:
nit: 10 minutes (just to keep the format the same)

Reply:
:+1: